speech/unit_tests/test_client.py

```diff
@@ -159,19 +173,43 @@ def test_sync_recognize_source_uri_without_optional_parameters(self):
         self.assertEqual(response[0].confidence, expected.confidence)
 
     def test_sync_recognize_with_empty_results(self):
+        from google.cloud._testing import _Monkey
+        from google.cloud.speech import client as MUT
         from google.cloud import speech
         from google.cloud.speech.sample import Sample
         from unit_tests._fixtures import SYNC_RECOGNIZE_EMPTY_RESPONSE
 
         credentials = _Credentials()
         client = self._makeOne(credentials=credentials)
         client.connection = _Connection(SYNC_RECOGNIZE_EMPTY_RESPONSE)
+        client._use_gax = False
 
-        with self.assertRaises(ValueError):
-            sample = Sample(source_uri=self.AUDIO_SOURCE_URI,
-                            encoding=speech.Encoding.FLAC,
-                            sample_rate=self.SAMPLE_RATE)
-            client.sync_recognize(sample)
+        with self.assertRaises(IndexError):
+            with _Monkey(MUT, _USE_GAX=False):
+                sample = Sample(source_uri=self.AUDIO_SOURCE_URI,
+                                encoding=speech.Encoding.FLAC,
+                                sample_rate=self.SAMPLE_RATE)
+                client.sync_recognize(sample)
+
+    def test_sync_recognize_with_gapic(self):
+        from google.cloud import speech
+        from google.cloud.speech import _gax as MUT
+        from google.cloud._testing import _Monkey
+        creds = _Credentials()
+        client = self._makeOne(credentials=creds)
+        client.connection = _Connection()
+        client._use_gax = True
+        client._speech_api = None
+
+        with _Monkey(MUT, RecognitionConfig=_RecognitionConfig,
+                     RecognitionAudio=_RecognitionAudio,
+                     SpeechApi=_MockGAPICSpeechAPI):
```

Comment:
They weren't available before because I hadn't added the the gapic and grpc packages to the setup.py REQUIREMENTS. Added now so these should be able to go away.